src/renderer/lib/sound.js

```diff
@@ -19,19 +19,19 @@ const sounds = {
   },
   DELETE: {
     url: 'file://' + path.join(config.STATIC_PATH, 'sound', 'delete.wav'),
-    volume: VOLUME
+    volume: VOLUME * 0.8
```

Comment:
@dcposch You're right. The delete sound is unchanged, so it is now louder than before. Will fix.